docs/docs/recipes/transforming-data.md

```diff
@@ -54,3 +54,65 @@ export const query = graphql`
 
 - [Tutorial on transforming Markdown to HTML](/tutorial/part-six/#transformer-plugins) using `gatsby-transformer-remark`
 - Browse available transformer plugins in the [Gatsby plugin library](/plugins/?=transformer)
+
+## Transforming images into grayscale using GraphQL
+
+### Prerequisites
+
+- A Gatsby site with `gatsby-config.js` and an `index.js` page
+- The `gatsby-transformer-sharp` plugin installed
+- The `gatsby-plugin-sharp` plugin installed
+- A source plugin installed, such as `gatsby-source-filesystem`
+- An image (`.jpg`, `.png`, `.gif`, `.svg`, etc.) in the `src` folder
+
+### Directions
+
+1. Make sure that the config file has the correct information.
+
+```JavaScript:title=gatsby-config.js
+
+ plugins: [
+   {
+     resolve: `gatsby-source-filesystem`,
+     options: {
+       name: `images`,
+       path: `${__dirname}/src/images`,
+     },
+   },
+   `gatsby-transformer-sharp`,
+   `gatsby-plugin-sharp`,
+ ],
+```
+
+Note: Note the gatsby src system path
+
+2.
+
+```query {
+     file(relativePath: { eq: "gatsby-astronaut.png" }) {
+       childImageSharp {
+         fluid(grayscale: true) {
+           ...GatsbyImageSharpFluid
+         }
+       }
+     }
+   }
+```
+
+Note: The relative path is the path that you config in gatsby src system.
+
+3. Next import image from "gatsby-image" line at the top
+
+ <Img fluid={data.file.childImageSharp.fluid}/>
+
+Note: `gatsby-image` is not a plugin. It gets installed but is just a component that you import. `transformer-sharp` and `plugin-sharp` have to be included as plugins.
+
+4. Run `gatsby develop` to start the development server.
+
+5. View your image in the browser: `http://localhost:8000/`
+
+### Additional resources
+
+- [API docs, including grayscale and duotone query tips](/docs/gatsby-image/#shared-query-parameters)
+- [Gatsby Image docs](/packages/gatsby-image/)
```

Comment:
The actual doc on Gatsby Image is probably a better starting point than the package README, since it pulls together information from all of the additional necessary plugins (`gatsby-plugin-sharp` and `gatsby-transformer-sharp`) that had information scattered around before. That's a workflow we went through recently on the learning team:
```suggestion
- [Gatsby Image docs](/docs/gatsby-image/)
```